woocommerce/woocommerce-ios
# [Woo POS] Floating control for menu and reader status

Closes: #13338


## Description


Added basic implementation of floating control for POS menu and reader status. p91TBi-bHy-p2

Floating control consists of 2 parts:
- "⋯" button that shows menu items "Get Support" and "Exit POS"
- Card reader status info view that shows card read connection status and if reader is not connected it is a button which starts the connection process

**Floating control overlaps some of the UI still and that will be updated as we update other UI parts of the UI to M2 design**

## Testing information


- Enter POS
- Use floating control to connect to card reader by tapping "Connect your reader" button
- Make sure that the reader is connected
- Add few products to cart
- Tap Checkout
- Finish the payment
- Tap "⋯" button in floating control and tap "Exit POS"

## Demo video


https://github.com/user-attachments/assets/41f5b03d-2e22-477a-a008-4f8c0df1c618

---
